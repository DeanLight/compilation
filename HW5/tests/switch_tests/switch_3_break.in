void main(){
	int x = 12;
	switch(x){
		case 11:
		printi(1);
		case 12:
		printi(2);
		case 13:
		printi(3);
		break;
		printi(0); 
		x = x+2;
		case 14:
		printi(4);
	};
}