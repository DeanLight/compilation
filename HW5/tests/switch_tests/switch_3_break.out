Loaded: ./exceptions.s
23